haystack/nodes/preprocessor/preprocessor.py

```diff
@@ -95,7 +98,9 @@ def __init__(
         self.split_length = split_length
         self.split_overlap = split_overlap
         self.split_respect_sentence_boundary = split_respect_sentence_boundary
-        self.language = iso639_to_nltk.get(language, language)
+        self.tokenizer_model_folder = tokenizer_model_folder
+        self.language = language
+        self.language_name = iso639_to_nltk.get(self.language, self.language)
```

Comment:
I was trying to avoid having to convert it back again. 

The custom model would be stored like `en.pickle.` This way, chances of bad typo (user saving model with a slightly different name than expected) would be minimized. Since it's an optional parameter and on other methods it can be `None`, we would not be able to get data back from Dict without doing some dictionary search. Do you think the dict search approach is better than an extra variable ?